images/scorecard-test/cmd/test/main.go

```diff
@@ -79,3 +90,19 @@ func main() {
 	fmt.Printf("%s\n", string(prettyJSON))
 
 }
+
+func printValidTests() (result v1alpha2.ScorecardTestResult) {
```

Comment:
```suggestion
// printValidTests will print out full list of test names to give a hint to the end user on what the valid tests are
func printValidTests() (result v1alpha2.ScorecardTestResult) {
```

Reply:
I updated the code with the function level comment you offered. thanks.